examples/decoding/plot_decoding_xdawn_transformer.py

```diff
@@ -1,50 +1,77 @@
 """
 ====================================================================
-Minimal example for using scikit-learn pipeline
+Decoding in sensor space data using the Xdawn Transformer
 ====================================================================
 
-This example creates a scikit-learn pipeline with mne transformers.
+Decoding applied to MEG data in sensor space decomposed using Xdawn.
+Here the classifier is applied to features extracted on Xdawn filtered signals.
 
 """
 # Authors: Asish Panda <asishrocks95@gmail.com>
+#          Alexandre Barachant <alexandre.barachant@gmail.com>
 #
 # License: BSD (3-clause)
+import numpy as np
 
-from sklearn.cross_validation import cross_val_score
+from sklearn.cross_validation import cross_val_score, ShuffleSplit  # noqa
 from sklearn.pipeline import make_pipeline
 from sklearn.linear_model import LogisticRegression
 from sklearn.preprocessing import label_binarize
+from sklearn.preprocessing import StandardScaler
 
 from mne import io, pick_types, read_events, Epochs
+from mne.viz import plot_topomap
 from mne.datasets import sample
 from mne.decoding import Vectorizer
 from mne.preprocessing import XdawnTransformer
 
-data_path = sample.data_path()
+import matplotlib.pyplot as plt
 
+data_path = sample.data_path()
 
 raw_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw.fif'
 event_fname = data_path + '/MEG/sample/sample_audvis_filt-0-40_raw-eve.fif'
-tmin, tmax = -0.1, 0.3
+tmin, tmax = -0.2, 0.5
 event_id = dict(aud_l=1, vis_l=3)
 
 # Setup for reading the raw data
 raw = io.read_raw_fif(raw_fname, preload=True)
-raw.filter(1, 20, method='iir')
+raw.filter(2., None, method='iir')
 events = read_events(event_fname)
 
-picks = pick_types(raw.info, meg=False, eeg=True, stim=False, eog=False,
+picks = pick_types(raw.info, meg='mag', eeg=False, stim=False, eog=False,
```

Comment:
This is the same for the CSP example (but using 'mag' instead of 'grad'). All type of sensor give the same results (99.6%), including EEG.